ChangeLog.md

```diff
@@ -4,21 +4,198 @@ Changelog
+- Rebalanced some bonuses from techs and specials to make high population less important and mitigate runaway empire growth "snowballing"
+
+
 ### Detailed Changelog
 
-#### Removed features
+#### Graphics / Interface
+
+- Added combat forces accordion panels to show more details
+- Zooming to an object as a chat command can look up an object by ID (as well as name)
+- Added arrived-from-system and ship production cost to objects list columns
+- Added a mouseover indicator for fields that indicates their size
+- Improved pedia search for entered text, which should now find more results, including articles generated at runtime
+- Made dark red and dark blue empire colours a bit lighter, to be easier to see on black backgrounds
+- Tweaked ship design screen layout
+- Ship design screen description reworked to be toggled active and multiline
+- Improved objects list refresh responsiveness
+- Made objects list column default wider
+- Added outer border highlighting of GUI window with input focus
+- Added widget for bold font texture file in options window
+- Moved font options to separate tab
+- Added vertical scale labels to empire statistic graphs
+- Added a right-click popup menu to empire statistic graphs, with several display options
+- Made the window resize tab bigger
+- Set a minimum size for some windows to prevent them from being made too small to be usable
+- Reworked pedia window layout, putting search bar and navigation buttons at top, similar to a web browser standard layout
+- Fixed display of circle arrow icons on production screen
+- Made options.xml contain only modified options by default, but added a button to output the full options database to the options window
+- Reworked splash / menu screen layout
+- Added time to produce at the currently selected location with available PP to production item tooltips
+- Added an option to reset empire readiness when joining a game, so orders can be revised even after they were submitted
+- Replace total cost and minimum production time with turns to produce, if fully funded, at current production location to the producible items list
+- Added indication in tooltips when production cost or minimum time are location-dependent and no location is selected
+- Added option to disable disclosure of galaxy seed
+- Prohibit typing some characters into galaxy seed text box, to prevent pedia layout script injection
+- The server stop auto-advancing turns after a player has won the game
+- Relabelled Resign button to Exit to Menu
+- Removed unused / redundant keymaps configuration file
+- Repeatedly clicking to close the game window will override the default autosave and exit faster
+
+#### Content and Balance
+
+- New Stuff
+    - Added Arc Disruptor ship weapon part
+    - Added sitreps for gifting to allied empires
+
+- Reworked what weapons can target fighters, ships, and planets
+    - Mass drivers, Lasers, Plasmas, Death Rays, and Spinal antimatter cannon attack ships and planets (no fighters)
```

Comment:
`cannon attack ships and planets (no fighters)`

What can be attacked and what not?   Looks like everything cannot be attacked by these weapons.

Github markdown allows tables.  May be we should create a table:  `[weapon, can attack ships, can attack planet can attack any type of fighter]`?

Reply:
I don't understand the question... those weapons attack ships and planets, not fighters. Maybe no vs not was unclear? I can remove the (no fighters) part...

Not keen on assuming this text is only to be read on GitHub and thus using its markup.

Reply:
> I don't understand the question...
 
Sorry I misread cannon and cannot.   Let it be as is.